### PR TITLE
add sudo in the bash command for running meshery

### PR DESCRIPTION
**Pull request by nupur-thakur**
Signed-off-by: nupur-thakur <nupur.s.thakur@gmail.com>

### CODE REVIEW

**Comment by anandrkskd** [comment]
@nupur-thakur did you updated Readme.md?

File: _includes/meshery.html [hunk 1]
```diff
@@ -207,7 +207,7 @@ <h4>Bash user</h4>
       <div class="highlight highlight-source-shell">
       <div style="width:auto; height:auto; padding:0px; position:relative;">
             <pre><code>
- $ curl -L https://git.io/meshery | bash -    
+ $ curl -L https://git.io/meshery | sudo bash -
```
Comment (leecalcote) [inline _includes/meshery.html]: It's not obvious, but you'll want to add `sudo` to [line 216](https://github.com/layer5io/layer5/pull/286/files#diff-a25801561d5b4546bebda5bdc902bf53L216) as well. This line defines what gets copied into the user's clipboard when clicking the copy button.

**Comment by leecalcote** [comment]
> @nupur-thakur did you updated Readme.md on purpose?

Good catch. @nupur-thakur you may need to sync your local fork (and branch) with new changes from the remote upstream (this repo on GitHub).

File: _includes/meshery.html (hunk 1 above)
Comment (nupur-thakur) [reply _includes/meshery.html]: Okay @leecalcote,  I completely missed this one. Thanks I'll update it now.

**Comment by nupur-thakur** [comment]
> @nupur-thakur did you updated Readme.md on purpose?

Yes. Because @girishranganathan suggested changes in 3 files including this one.

**Comment by nupur-thakur** [comment]
Just realised the error in my PR was fixed in the PR by #308

**Pull request closed without merging**